CONTRIBUTING.md

```diff
@@ -20,30 +20,36 @@ contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additio
     - Rust: `// Copyright (c) Microsoft. All rights reserved.`
     - C: `/* Copyright (c) Microsoft. All rights reserved. */`
 
-    This is enforced by `make test`
+    This is enforced by `make test`.
+
+1. Rust code must be formatted with `rustfmt`
+
+    This is enforced by `make test-release`.
+
+    When developing locally, consider using `make test` instead, which does not enforcing rustfmt code formatting.
```

Comment:
"enforcing" -> "enforce"

Also, probably something like:

```diff
+But make sure to run `make test-release` before you push to your remote so that you run all the checks that CI would run.
```